### PR TITLE
add deploy lightning

### PR DESCRIPTION
**Pull request by liubo0127**
add deploy lightning
@LinuxGit @superlzs0476 PTAL

### CODE REVIEW

**Comment by kennytm** [comment]
Seems missing the checkpoint config...

File: conf/tikv-importer.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+import:
+  # this directory is used to store the data written by `tidb-lightning`.
+  # import-dir: "/tmp/tikv/import"
+  # the number of threads to handle RPC requests.
+  num-threads: 16
+  # the number of concurrent import jobs.
+  num-import-jobs: 24
+  # the stream channel window size. Stream will be blocked when the channel is full. 
+  stream-channel-window: 128
```
Comment (kennytm) [inline conf/tikv-importer.yml]: This missed the `max-open-engines` setting

https://github.com/tikv/tikv/blob/6ff5c91e08d2bad9cf3d4ffbb4120459d814585a/etc/tikv-importer.toml#L53

File: roles/local/tasks/main.yml [hunk 2]
```diff
@@ -171,6 +192,11 @@
     - svc
   when: process_supervision == 'supervise'
 
+- name: cp tidb binary
```
Comment (LinuxGit) [inline roles/local/tasks/main.yml]: cp lightning binary

File: roles/tidb_lightning/tasks/binary_deployment.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    src: "{{ item }}_lightning_binary.sh.j2"
+    dest: "{{ deploy_dir }}/scripts/{{ item }}_lightning.sh"
+    mode: "0755"
+    backup: yes
+  with_items:
+    - start
+    - stop
+  register: lightning_script
+
+- name: backup script file
```
Comment (LinuxGit) [inline roles/tidb_lightning/tasks/binary_deployment.yml]: Maybe lightning_script.backup_file is always undefined, you can use debug module to test it.
refer to https://github.com/pingcap/tidb-ansible/blob/master/roles/prometheus/tasks/main.yml#L31

File: roles/tikv_importer/tasks/binary_deployment.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    src: "{{ item }}_importer_binary.sh.j2"
+    dest: "{{ deploy_dir }}/scripts/{{ item }}_importer.sh"
+    mode: "0755"
+    backup: yes
+  with_items:
+    - start
+    - stop
+  register: importer_script
+
+- name: backup script file
```
Comment (LinuxGit) [inline roles/tikv_importer/tasks/binary_deployment.yml]: ditto

**Pull request merged**